src/otx/recipe/_base_/data/classification.yaml

```diff
@@ -55,7 +65,7 @@ val_subset:
   transforms:
     - class_path: otx.core.data.transform_libs.torchvision.Resize
       init_args:
-        scale: $(input_size)
+        scale: [256, 256]  # Updated input size
```

Comment:
please, use $(input_size). It will be automatically set to the right input size based on the provided input size to the model. You need to change this only on the 3-4 lines